modules/EvseManager/Charger.hpp

```diff
@@ -80,14 +82,16 @@ class Charger {
     float getMaxCurrent();
     sigslot::signal<float> signalMaxCurrent;
 
-    enum class ChargeMode {
+    enum class ChargeMode
+    {
         AC,
         DC
     };
 
     void setup(bool three_phases, bool has_ventilation, const std::string& country_code, bool rcd_enabled,
```

Comment:
this function has way too many parameters and gets like 5 bools in row which you can easily scramble up by accident, same for the two floats.  Should be refactored!

Reply:
This should be refactored, was not touched in this PR